tests/model/test_model.py

```diff
@@ -434,6 +434,28 @@ def test_react_to_message_for_not_thumbs_up(self, model):
         with pytest.raises(AssertionError):
             model.react_to_message(dict(), 'x')
 
+    def test_send_typing_event_true(self, mocker, model,
+                                    recipient_user_ids=[1]) -> None:
+        mock_api_query = mocker.patch('zulipterminal.core.Controller'
+                                      '.client.set_typing_status')
+
+        model.send_typing_event(recipient_user_ids, True)
+
+        mock_api_query.assert_called_once_with(
+            {'to': recipient_user_ids, 'op': 'start'},
+        )
+
+    def test_send_typing_event_false(self, mocker, model,
+                                     recipient_user_ids=[1]) -> None:
+        mock_api_query = mocker.patch('zulipterminal.core.Controller'
+                                      '.client.set_typing_status')
+
+        model.send_typing_event(recipient_user_ids, False)
+
+        mock_api_query.assert_called_once_with(
+            {'to': recipient_user_ids, 'op': 'stop'},
+        )
```

Comment:
I will go through `pytest.mark.parametrize` in more detail and see if I can simplify the tests further. Initially, at first glance, I didn't consider `pytest.mark.parametrize` because I saw tests similar to the current iteration of this test for asynchronous `model.py` functions.

Reply:
`pytest.mark.parametrize` seems to reduce the need for two tests here, I'll use that here.